changelog/v1.25.1-patch3/request-transform-improvements.yaml

```diff
@@ -0,0 +1,9 @@
+changelog:
+- type: NEW_FEATURE
+  issueLink: https://github.com/solo-io/solo-projects/issues/4571
+  resolvesIssue: false
+  description: >
+    Adds support for queryStringParameters, multiValueQueryStringParameters, and
+    multiValueHeaders to the requestTransformation feature in the transformation
+    filter. This feature is leveraged by the requestTransformation setting in our
+    AWS Lambda filter.
```

Comment:
i think we should also update the proto comment to reflect the new fields added as well https://github.com/solo-io/envoy-gloo/blob/74cea33874f2bd0ec4f10c7590023c6d2ff53689/api/envoy/config/filter/http/transformation/v2/transformation_filter.proto#L265-L267